packages/admin/src/Resources/Pages/ListRecords.php

```diff
@@ -239,35 +277,30 @@ protected function getMountedActionFormModel(): string
         return $this->getModel();
     }
 
-    public function getTableRecordTitle(Model $record): ?string
+    public function getTableRecordTitle(Model $record): string
```

Comment:
Changing the return type from `?string` to `string` is a breaking change

Reply:
This isn't a documented method and the main config for record titles is in the resource, not this class

Reply:
Just thought that people might have overwritten it.